core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -84,30 +134,131 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
       this.isTranslationCoordinator = userInfo.isTranslationCoordinator();
 
       if (this.isTranslationCoordinator) {
-        this.CONTRIBUTION_TYPES = [this.TAB_NAME_TRANSLATION_SUBMITTER,
-          this.TAB_NAME_TRANSLATION_REVIEWER];
+        this.CONTRIBUTION_TYPES.push(
+          this.TAB_NAME_TRANSLATION_SUBMITTER,
+          this.TAB_NAME_TRANSLATION_REVIEWER);
+        this.contributorDashboardAdminStatsBackendApiService
+          .fetchAssignedLanguageIds(username).then(
+            (response) => {
+              this.assignedLanguageIds = response;
+              this.languages = this.languages.filter((
+                  languageItem) =>
+                this.assignedLanguageIds.includes(languageItem.id));
```

Comment:
Is this the only place `assignedLanguageIds` is used? If so, then let's get rid of it as a class instance variable.

Reply:
removed the variable as a whole, we can directly use response instead.